### ISSUE TITLE
Declutter markers in new implementation

### ISSUE THREAD

**Issue by IanMayo** [serge-web/serge#441]
## 💬 Brief description of feature

The legacy mapping had a feature to declutter markers.

Currently markers (asset icons, turn waypoint markers) as displayed over each other, at the centre of the relative hex cell.

After map changes, we should declutter the markers.

We're in a very difficult architecture now.   But, all of the asset locations and turn marker locations are present at the same time in `assets\index.tsx` - so that looks like the logical place to conduct the decluttering.

It's difficult to demonstrate this issue. But, in the following animation we see that `T06` and `T07` are present on two tracks.

![Before_Declutter](https://user-images.githubusercontent.com/1108513/84285825-d85fe100-ab35-11ea-87b9-7a75a37f14cc.gif)

## 📖 User Story

* When I view the Serge map Asset icons and Turn markers that are in the same cell are distributed around that cell.

## ☑️ Acceptance criteria

- [ ] Tests pass

**Comment by IanMayo** [comment]
@lilitkarapetyan  - I started this, but then gave up almost immediately.

I _think_ the place for declutter is in the `asset\index.tsx` file. It's this location where the asset icons and the route markers are created.

I _guess_ that once the React Leaflet elements have been created, we need to access the Leaflet control, find the layer containing the icons and markers, and then run declutter on their locations.

Note: the legacy way of running declutter was in `declutterLayer.js`.  I wrote that code, so I'm quite familiar with it.  I'm happy to help out - but you're the expert in getting at the Leaflet data.

Hold on. I now realise there's another strategy. We declutter all of the locations before we create the React elements.  This actually seems easier. I just did `inspect` on one of the route lines, and the positions are represented in quite scary SVG:
![Screen Shot 2020-08-18 at 17 24 52](https://user-images.githubusercontent.com/1108513/90539475-c45cdf00-e177-11ea-8dde-887c52195d55.png)

Ok.  Based on the second strategy - I'll give this a go myself.
